code/game/say.dm

```diff
@@ -71,7 +71,7 @@ GLOBAL_LIST_INIT(freqtospan, list(
 	return ""
 
 /atom/movable/proc/compose_job(atom/movable/speaker, message_langs, raw_message, radio_freq)
-	return "[" (" + speaker.GetJob() + ")"]"
+	return "[ speaker.GetJob() ? " (" +  speaker.GetJob() + ")" : ""]"
```

Comment:
I would suggest instead of calling the GetJob proc twice, to make it into a var to reference twice.

Reply:
Sorry I have a single-line mentality